libraries/chain/include/graphene/chain/protocol/sidechain_transaction.hpp

```diff
@@ -26,7 +28,7 @@ namespace graphene { namespace chain {
 
       asset                         fee;
       account_id_type               payer;
-      proposal_id_type              proposal_id;
+      bitcoin_transaction_id_type   tx_id;
```

Comment:
I'm not sure really is it correct way or not. 
SONs can verify every set of signatures separately or verify them all at the end, and they should verify them at the end in any case,  so I have decided to gather signatures without verification and verify all of them at the end.

Reply:
If we delink the proposal approval process from the signing procedure, there will be a delay in signature collection. There will be one extra step(solitary approval of send proposal) now with these new changes.
Earlier:
send -> sign by multiple sons and proposal approval in one go -> process transaction object and send out the signed tx.

Now:
send -> proposal approval by multiple sons -> signing by multiple sons -> process transaction object and send out the signed tx.

Reply:
Yes,  you are right,  I've reverted it back to proposal already.) I'm not sure it works correctly when inactive SONs should sign the tx (during the primary wallet recreation), I'll try to write some tests for this case.